File: Loops_Lucho_Stoymirov/fizzbuzz.js


let fizzBuzz = number =>{
    var x = number;
    var result = [];
    for (var i = 1; i <= x; i++) {
        if (i % 3 ==0 && i % 5 != 0) {
            result.push("Fizz");
        } else if (i % 5 ==0 && i % 3 != 0) {
            result.push("Buzz");
        } else if (i % 3 == 0 && i % 5 == 0) {
            result.push("FizzBuzz");
        } else {
            result.push(i);
        }
    }
    console.log(result);
}

fizzBuzz(15);

File: Loops_MomchilSlavov/First Project/FizzBuzz.cs
/*
 * C# Program aimed at displaying "Fizz", "Buzz" or "FizzBuzz" based on the division of a certain number
 */

public class For_Loop
{
    public static void main(String[] args)
    {
        int n = 15;
        for (int i = 1; i <= 15; i++)
        {
            if (n % 3 == 0 && n % 5 == 0)
            {
                Console.WriteLine("FizzBuzz");
            }
            //System.out.println(i);
            else if (n % 3 == 0)
                Console.WriteLine("Fizz");
            else if (n % 5 == 0)
                Console.WriteLine("Buzz");
        }
    }
}


   





    